writer.go

```diff
@@ -242,6 +251,7 @@ func (p *MasterPlaylist) Version() uint8 {
 // automatically by other Set methods.
 func (p *MasterPlaylist) SetVersion(ver uint8) {
 	p.ver = ver
+	p.dirty = true
```

Comment:
Does this need to be protected by the lock ?

Reply:
No, nothing can broke here if left unprotected.